src/result.js

```diff
@@ -127,29 +127,25 @@ function missingTag(tag) {
 
 function insertIntoIndexHTML(html, head, body) {
   if (!html) { return Promise.resolve(html); }
+  let isBodyReplaced = false;
+  let isHeadReplaced = false;
 
-  if (body) {
-    let isBodyReplaced = false;
-    html = html.replace("<!-- EMBER_CLI_FASTBOOT_BODY -->", function() {
+  html = html.replace(/<\!-- EMBER_CLI_FASTBOOT_(HEAD|BODY) -->/g, function(match, p1) {
+    if (p1 === 'HEAD' && head && !isHeadReplaced) {
+      isHeadReplaced = true;
+      return head;
+    } else if (p1 === 'BODY' && body && !isBodyReplaced) {
       isBodyReplaced = true;
       return body;
-    });
-
-    if (!isBodyReplaced) {
-      return missingTag('<!--EMBER_CLI_FASTBOTT_BODY-->');
     }
-  }
+    return '';
+  });
 
-  if (head) {
-    let isHeadReplaced = false;
-    html = html.replace("<!-- EMBER_CLI_FASTBOOT_HEAD -->", function() {
-      isHeadReplaced = true;
-      return head;
-    });
-
-    if (!isHeadReplaced) {
-      return missingTag('<!--EMBER_CLI_FASTBOTT_HEAD-->');
-    }
+  if (head && !isHeadReplaced) {
```

Comment:
https://github.com/bekzod/fastboot/blob/45087528437aa0d1c2dc620b9a45898a8063ce1a/src/result.js#L37-L39 because sometimes missing head insert comment tag is still valid, the same logic was used before, if I remove head bunch of test are broken